restcomm/restcomm.core/src/main/java/org/restcomm/connect/core/service/util/UriUtils.java

```diff
@@ -89,41 +94,56 @@ private static HttpConnectorList getHttpConnectors() throws Exception {
     }
 
     /**
-     * Resolves a relative URI.
+     *
+     * Use to resolve a relative URI
+     * using the instance hostname as base
+     *
+     * @param uri
+     * @return
+     */
+    public URI resolve(final URI uri) {
+        return resolve(uri, null);
+    }
+
+    /**
+     * Use to resolve a relative URI
+     * using the domain name of the Organization
+     * that the provided AccountSid belongs
      *
      * @param uri The relative URI
+     * @param accountSid The accountSid to get the Org domain
      * @return The absolute URI
      */
-    public static URI resolve(final URI uri) {
+    public URI resolve(final URI uri, final Sid accountSid) {
```

Comment:
i see that we removed UriUtilsTest, does it makes sense to cover this method in unit test?

Reply:
@maria-farooq The UriUtilsTest was almost empty, it had one empty test.
I plan to create new unit test for this